### DISCUSSION TITLE
Release 1.0.0 !

### DISCUSSION THREAD

**Discussion by Zavy86** [Zavy86/WikiDocs#62] [Announcements]
Finally, after years of hit and miss development, the first stable version of **Wiki|Docs** has been released.
I wanted to give a special thanks to all the _Contributors_ who, in addition to helping me in development, encouraged me to continue and not to abandon the project.
There is still a lot to do but we are on the right way..

![f89b1c7d38d8fc67d7199a11a10234cd](https://github.com/Zavy86/WikiDocs/assets/2051811/6098b980-878a-46f8-b9c8-8ed98ae7c132)

Thank you @leomoon, @jv3ga, @reyemxela, @serial, @bitwisecreative, @sam-6174, @kevwkev!

**Comment by zerosonesfun** [comment]
Amazing! Congratulations! 🎉

**Comment by jv3ga** [comment]
Awesome!!! Good job everyone!!!

**Comment by bitwisecreative** [comment]
Rad!
